service/src/main/java/org/folio/rest/workflow/service/WorkflowEngineService.java

```diff
@@ -47,11 +47,8 @@ public class WorkflowEngineService {
   @Value("${okapi.url}")
   private String okapiUrl;
 
-  @Value("${okapi.camunda.base-path}")
-  private String basePath;
-
-  @Value("${okapi.camunda.rest-path}")
-  private String restPath;
+  @Value("${okapi.workflow-engine.path:/camunda}")
```

Comment:
Update [here](https://github.com/folio-org/mod-camunda/pull/123/files#diff-1bad63b7b3b478da42579fc1f09e8700cf6c258922367ac29d1f64bd90b75d19R97) when mod-camunda module descriptor path pattern `camunda` is refactored to `workflow-engine`.